src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
Consider using an interface to define behavior for figures instead of extending from a concrete class, as this will provide a more flexible design allowing for the independent evolution of hierarchies.